.changeset/cyan-melons-hunt.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/memory': patch
+---
+
+Improved the reliability of experimental Subconscious memory behavior.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)